evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -781,25 +820,18 @@ where
     /// starting from that length, for each `degree_bits` in the range specified
     /// for this STARK module. Specifying a wide enough range allows a
     /// prover to cover all possible scenarios.
-    /// - `stark_config`: the configuration to be used for the STARK prover. It
-    ///   will usually be a fast one yielding large proofs.
+    /// - `config`: the configuration to be used for the STARK prover. It will
+    ///   usually be a fast one yielding large proofs.
```

Comment:
```suggestion
    /// - `config`: the set of configurations to be used for the different proof layers. It will
    ///   usually be a fast one yielding large proofs for the base STARK prover, and a more
    /// flexible one for the recursive layers.
```